src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -16,30 +17,76 @@ const adminCondition = [
 	}
 ];
 
-const modules = {
-	jiraDevelopmentTool: {
-		application: {
-			value: "GitHub"
-		},
-		capabilities: [
-			"branch",
-			"commit",
-			"pull_request"
-		],
-		actions: {
+interface JiraDevelopmentToolActions {
+	createBranch?: {
+		templateUrl: string;
+	};
+	searchConnectedWorkspaces?: {
+		templateUrl: string;
+	};
+	searchRepositories?: {
+		templateUrl: string;
+	};
+	associateRepository?: {
+		templateUrl: string;
+	};
+}
+
+const CREATE_BRANCH_ENDPOINT =
+	`${envVars.APP_URL}/create-branch-options?issueKey={issue.key}&issueSummary={issue.summary}&tenantUrl={tenant.url}&jwt={jwt}&addonkey=${envVars.APP_KEY}`;
+const SEARCH_CONNECTED_WORKSPACES_ENDPOINT = `${envVars.APP_URL}/jira/workspaces/search`;
+const SEARCH_REPOSITORIES_ENDPOINT = `${envVars.APP_URL}/jira/workspaces/repositories/search`;
+const ASSOCIATE_REPOSITORY_ENDPOINT = `${envVars.APP_URL}/jira/workspaces/repositories/associate`;
+
+export const getGenericContainerUrls = async (): Promise<string[] | null> => {
```

Comment:
Other than newly installed, expect up to 24hours of delays once you flick this flag

Reply:
You sure? It was pretty instantaneous for me in dev/stg.